### PR TITLE
Feature: Add spans pip package

### PR DESCRIPTION
**Pull request by plieningerweb**

### CODE REVIEW

File: recipes/spans/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{% set version = "0.3.0" %}
+{% set sha256= "d17d36dc240525e97eae26ff68b63b8a9dc6581e4668581f6e189c24ea50f201" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/74/73/2d0192f38a8caad9ab439210ef943825312902cc9280c9b73a196b5be719/Spans-0.3.0.tar.gz
```
Comment (jakirkham) [inline recipes/spans/meta.yaml]: Please use the URL from the [`example`]( https://github.com/conda-forge/staged-recipes/blob/6ca29ae558f4353a108a26f7ffb23250a021bea7/recipes/example/meta.yaml#L18 ).

**Comment by jakirkham** [comment]
After changing the URL, LGTM.

**Comment by plieningerweb** [comment]
I could not find the same URL as in the example, using the filename.

Can you help me?

File: recipes/spans/meta.yaml (hunk 1 above)
Comment (synapticarbors) [inline recipes/spans/meta.yaml]: You should set the name to:

```
{% set name = "Spans" %}
```

(note the capitalization). Then the url would be:

```
url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

as in: 

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

File: recipes/spans/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  summary: 'Continuous set support for Python'
+
+  description: |
+    Spans is a pure Python implementation of PostgreSQL’s range types [1]. Range types are conveinent when working with intervals of any kind. Every time you’ve found yourself working with date_start and date_end, an interval may have been what you were actually looking for.
+  doc_url: http://spans.readthedocs.org/en/latest/
+  dev_url: https://www.github.com/runfalk/spans
+
+extra:
+  recipe-maintainers:
+    - runfalk
```
Comment (synapticarbors) [inline recipes/spans/meta.yaml]: The recipe maintainer should be you, not the author of the package. You are welcome to contact the package author and ask them if they want to be included, but we don't want to add people without their expressed permission.

Comment (synapticarbors) [inline recipes/spans/meta.yaml]: @plieningerweb -- any updates to this? Also when you do update this PR, please pull in the latest version of master.

**Pull request closed without merging**